<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiriadPlotter</class>
 <widget class="QWidget" name="MiriadPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Miriad Plotter</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="15,1">
    <property name="spacing">
     <number>8</number>
    </property>
    <item>
     <widget class="QFrame" name="plotWidget">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <zorder>horizontalLayoutWidget</zorder>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="dmSlider">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="dmLabel">
        <property name="text">
         <string>DM: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="plotterResources.qrc"/>
 </resources>
 <connections/>
</ui>
